src/components/PrintContent/PrintContent.js

```diff
@@ -7,12 +7,16 @@ import { buildTemplate } from '../../utils';
 
 import css from './PrintContent.css';
 
-const PrintContent = forwardRef(({ dataSource, template }, ref) => {
+const PrintContent = forwardRef(({
+  dataSource,
+  template,
+  printContentTestId,
```

Comment:
Let's just call this prop `id` - it seems weird that we have a prop that is designed not for functionality, but for tests
Also, let's just make it optional, since functionality doesn't change by this prop - only tests are affected